src/main/java/net/dv8tion/jda/core/entities/EntityBuilder.java

```diff
@@ -1286,7 +1286,7 @@ else if (channelType.isGuild())
 
     public ApplicationInfo createApplicationInfo(JSONObject object)
     {
-        final String description = object.getString("description");
+        final String description = object.optString("description", null);
```

Comment:
This should be `optString("description, "")`

Reply:
Wish granted